cha/implementaion/playback/meta.tex

```diff
@@ -0,0 +1,28 @@
+In this chapter we explore some key elements in the development of our application related to the playback of audio data.
+The chapter includes our decisions, the reasoning behind them, some of the complications we encountered and how we dealt with them.
+Before we present the implementation and what that entails, we firstly present some audio data terminology, as well as the Android sound stack, as this domain knowledge is used in our implementation.
+
+\begin{description}
+    \item [Sample Rate]\cite{sample_rate_std} \hfill \\
+    Sample rate is the number of audio samples per channel per second.
+    The rate may vary from file to file with $44,100\ Hz$ being the de facto and most supported sampling rate.
```

Comment:
In the source I have sourced it mentions 44100 is refered to as de facto standard